tests/eko/kernels/test_kernels_QEDns.py

```diff
@@ -159,7 +159,7 @@ def test_ode():
                     evmod,
                     masses,
                     hqm_scheme=QuarkMassSchemes.POLE,
-                    thresholds_ratios=MatchingScales(c=1.0, b=1.0, t=1.0),
+                    thresholds_ratios=None,
```

Comment:
I believe this should be
```suggestion
                    thresholds_ratios=[1., 1., 1.],
```
and below of course (please check yourself)